### DISCUSSION TITLE
Is it possible to restrict temporary emails?

### DISCUSSION THREAD

**Discussion by ikupenov** [logto-io/logto#4605] [Q&A]
I was wondering whether it would be possible to restrict temporary emails or perhaps have a whitelist of allowed email domains? In some cases, it'd also be nice to have the option to allow only company email domains.

**Comment by gao-sun** [comment]
this is a good proposal. cc @Rany0101 @fleuraly

**Comment by Rany0101** [comment]
The "Block disposable email addresses" feature has been released. See: https://docs.logto.io/security/blocklist#block-disposable-email-addresses
